controller/array_action/array_mediator_ds8k.py

```diff
@@ -44,6 +46,9 @@
 ARRAY_SPACE_EFFICIENCY_THIN = ds8k_types.DS8K_TP_ESE
 ARRAY_SPACE_EFFICIENCY_NONE = ds8k_types.DS8K_TP_NONE
 
+volume_cache = dict()
```

Comment:
I think this should be per management address, in case there is more than 1.
i.e. per mediator instance.
it should work since we also cache the mediator instances per management address

Reply:
no, didn't worked

Reply:
you mean the mediator cache is missing hits? which part didn't work?

Reply:
the cache is always empty like it creates a new one every time

Reply:
sounds like a bug in the mediator cache.
is the ds8k volume id universally unique? or can it be the same on separate ds8k systems?

Reply:
done